assertk/src/jvmMain/kotlin/assertk/assertions/any.kt

```diff
@@ -75,6 +75,7 @@ fun <T : Any> Assert<T>.isNotInstanceOf(jclass: Class<out T>) = given { actual -
  * assertThat(person).prop(Person::name).isEqualTo("Sue")
  * ```
  */
+@Deprecated("Use an overload with KProperty1 instead")
```

Comment:
Am I correct in thinking that existing code would automatically move to the `KProperty1` overload? If that's the case then anything still calling the deprecated method would not be a `KProperty1`. Would have the deprecation message point them to the `prop(name, extractor)` method instead.

Reply:
Agreed

Reply:
It is a shame that dokka still does not support overload references 😔